pyacal/devices/base.py

```diff
@@ -162,7 +162,7 @@ def wait(
         """."""
 
         def comp_(val):
-            boo = comp(self[propty], val)
+            boo = comp(getattr(self, propty), val)
```

Comment:
I think this is not how the code is intended to work. We need to use `self[propty]` in order to get the alias for the PV, not the python attribute associated with it. Changing this would break the existing code.